### PR TITLE
Project1

### PR DESCRIPTION
**Pull request by itseugene**
## Context for your design

​

... Write 3-4 sentences here about the audience for your design and how this affected your choices re: colors, typography, etc...

- The targeted readers of my design is people who do not like read. Therefore, the theme color of my design is a pastel tone of pink since it has a soft and friendly atmosphere which helps readers could enjoy the reading. To make a high legible screen, I chose the font called 'PT Sans(Sans-serif)'. The beginning of the first lines have bigger and heavy fonts that help users to read easily. When the time and place changes I used highlighters to give a notice that an episode is ended. In addition, small size of fonts represent negative voice of the author.

## HTML Best Practices

​

  * [ ]  HTML is indented to enhance readability

​

  * [ ]  HTML is lowercase

​

  * [ ]  Double quotes are used around attributes (e.g., `href="https://mica.edu"`, `class="class-name"`)

​

  * [ ]  Markup makes use of semantic elements and avoids `div`-itis

​

  * [ ]  A proper HTML5 `doctype` is set

​

  * [ ]  The HTML head includes a proper `title`

​

  * [ ]  Heading tags are used semantically, not simply for presentation

​

  * [ ]  Markup content is not presented in ALLCAPS for presentational purposes; this is achieved via the `text-transform` CSS property instead

​

  * [ ]  HTML is error free

​

​

## CSS Best Practices

​

  * [ ]  CSS is lowercase

​

  * [ ]  Declarations within each ruleset are ordered alphabetically by property name

​

  * [ ]  A semicolon ends each declaration

​

  * [ ]  Comments are used to break up sections and explain your code

​

  * [ ]  `!important` is avoided overall and, when used, is justified via a comment

​

  * [ ]  CSS does **not** reiterate browser defaults

​

  * [ ]  CSS is not written in such a way that it is immediately overwritten by other declarations

​

  * [ ]  Color values are used logically and consistently (e.g., hex and hsl values aren't mixed)

​

  * [ ]  Units are used logically and consistently

​

  * [ ]  Zero measurements do **not** include units

​

  * [ ]  CSS is error free

Message #general
*bold* _italics_ ~strike~ `code` ```preformatted``` >quote
Workspace Directory
EUGENE LEE
Status
Display name
EUGENE LEE
Local time
8:21 PM
Email address
elee05@mica.edu

### CODE REVIEW

File: project1/index.html [hunk 1]
```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+
+
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
```
Comment (angeliquejw) [inline project1/index.html]: Remember to update your `title` with something relevant to your page or site.

File: project1/index.html [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    <link href="https://fonts.googleapis.com/css?family=Caesar+Dressing|Metal+Mania|Nunito+Sans|Squada+One&display=swap"
+        rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=PT+Sans+Caption:700&display=swap" rel="stylesheet">
+
+</head>
+
+
+<body>
+    <main>
+        <div class="cut">
```
Comment (angeliquejw) [inline project1/index.html]: Instead of a `div` consider using a `header` element here:
```suggestion
        <header class="cut">
```

File: project1/index.html [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+            <h1><span class="Hlight">The Crane Wife</span></h1>
+            <h2>By CJ Hauser</h2>
+            <h3>July 16, 2019</h3>
+        </div>
```
Comment (angeliquejw) [inline project1/index.html]: To match the suggestion above:
```suggestion
        </header>
```

File: project1/index.html [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+
+        <p><span class="first">Ten days after</span> I called off my engagement I was supposed to go on a scientific
```
Comment (angeliquejw) [inline project1/index.html:27]: On line 43, you use a `b` element for this, which is appropriate -- but, even if you were going to keep it a `span`, you'd want this to be consistent on every paragraph.

File: project1/index.html [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+            home, reflecting on the enormity of what has transpired and not doing whatever it is I am about to be
+            doing that
+            requires a pair of plastic clogs with drainage holes. Surely, I thought, as I tried on a very large and
+            floppy
+            hat featuring a pull cord that fastened beneath my chin, it would be wrong to even be wearing a hat that
+            looks
+            like this when something in my life has gone so terribly wrong.</p>
+
+        <p>
+            <b class="first">Ten days <span class="letter">earlier</span></b> I had cried and I had yelled and I had
```
Comment (angeliquejw) [inline project1/index.html:43]: The "letter" class and the `span` here aren't doing anything different than the "first" class and the `b element?
```suggestion
            <b class="first">Ten days earlier</b> I had cried and I had yelled and I had
```

File: project1/index.html [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+            away
+            from
+            the
+            upstate
+            New
+            York house with two willow trees I had bought with my fiancé.
+        </p>
+
+        <p>
+            <div class="one">Ten days later and I didn’t want to do anything I was supposed to do.</div>
```
Comment (angeliquejw) [inline project1/index.html:54]: You don't need the `div` here -- you can just attach the classname to your `p` element, like so:
`<p class="one">`

File: project1/index.html [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+            people of
+            the Earthwatch organization assured me I was welcome on the trip and would get to participate in “real
+            science”
+            during my time on the gulf. But as I waited to be picked up by my team in Corpus Christi, I was
+            nervous—I
+            imagined everyone else would be a scientist or a birder and have daunting binoculars.</p>
+
+        <p><span class="first">The biologist</span> running the trip rolled up in in a large white van with a boat hitch
+            and the words
+            <span class="light"><b><i>BIOLOGICAL
```
Comment (angeliquejw) [inline project1/index.html:73]: 1. Review the [use cases for the `i` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/i). I don't think this is a valid use.
1. Instead of adding the `span`, you can add the "light" class to your `b` element:
```
<b class="light">
````

File: project1/index.html [hunk 8]
```diff
@@ -0,0 +1,137 @@
...
+            week
+            before. The first thing Jeff said was, <strong>“We’ll head back to camp, but I hope you don’t mind we run by
+                the
+                liquor
+                store first.”</strong> I felt more optimistic about my suitability for science.</p>
+
+        <div>*</div>
+
+        <p>
+            <div class="one">Not long before I’d called off my engagement it was Christmas.</div>
```
Comment (angeliquejw) [inline project1/index.html:89]: See previous note about dropping the `div` and adding the classname to the `p` element.

File: project1/index.html [hunk 9]
```diff
@@ -0,0 +1,137 @@
...
+            What she was offering was so nice. She was so nice. I thanked her and felt
+            ungrateful for having wanted a
+            stocking, but not this stocking. <div class="one">
+                <blockquote>Who was I to be choosy? To say that this nice thing
+                    she was
+                    offering
+                    wasn’t a
+                    thing I wanted?</blockquote>
+            </div>
+        </p>
```
Comment (angeliquejw) [inline project1/index.html:124]: The `blockquote` should appear separate from the `p` element here and can have the "one" classname attached to it, like so:
```html
        <p>What she was offering was so nice. She was so nice. I thanked her and felt ungrateful for having wanted a stocking, but not this stocking.</p>
        <blockquote class="one">Who was I to be choosy? To say that this nice thing she was offering wasn’t a thing I wanted?</blockquote>
```

File: project1/styles.css [hunk 10]
```diff
@@ -0,0 +1,157 @@
+@import url('https://fonts.googleapis.com/css?family=Noto+Sans|PT+Sans|Source+Sans+Pro|Ubuntu&display=swap');
```
Comment (angeliquejw) [inline project1/styles.css:1]:
```suggestion
```

Since you have the `link` referencing this stylesheet on your HTML, you don't need this import rule here.

File: project1/index.html [hunk 11]
```diff
@@ -0,0 +1,137 @@
...
+
+
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Caesar+Dressing|Metal+Mania|Nunito+Sans|Squada+One&display=swap"
```
Comment (angeliquejw) [inline project1/index.html:12]: If you are not using these fonts in your final project, you do not want to load them:
```suggestion
```

File: project1/styles.css [hunk 12]
```diff
@@ -0,0 +1,157 @@
...
+    margin: 40px 0 18px 0;
+    /* color: rgb(87, 2, 30); */
+
+}
+
+h2 {
+    font-size: 13px;
+    font-family: 'PT Sans', sans-serif;
+    text-align: center;
+    margin: 0 0 0 0;
```
Comment (angeliquejw) [inline project1/styles.css:39]: You can shorten this to a single zero, since the measurements are the same for each side of the box:
```suggestion
    margin: 0;
```

File: project1/styles.css [hunk 13]
```diff
@@ -0,0 +1,157 @@
...
+
+ */
+body {
+    /* background-color: white;
+    background-color: hsla(102, 34%, 50%, 0.3); */
+    background-color: hsla(0, 100%, 71%, 0.2);
+    color: black;
+    max-width: 800px;
+    margin: 0 auto;
+}
```
Comment (angeliquejw) [inline project1/styles.css:22]: You can also add your `font-family` rule to the `body` and then not have to repeat it on so many elements 👍🏻

File: project1/styles.css [hunk 14]
```diff
@@ -0,0 +1,157 @@
...
+h2 {
+    font-size: 13px;
+    font-family: 'PT Sans', sans-serif;
+    text-align: center;
+    margin: 0 0 0 0;
+    font-weight: 700;
+    letter-spacing: 1px;
+    /* color: rgb(87, 2, 30); */
+    color: rgb(48, 18, 28);
+    font-weight: bold;
```
Comment (angeliquejw) [inline project1/styles.css:44]: This rule is overriding the rule you have on line 40.

File: project1/styles.css [hunk 15]
```diff
@@ -0,0 +1,157 @@
...
+    font-family: 'PT Sans', sans-serif;
+    font-size: 13px;
+    text-align: center;
+    font-weight: 200;
+    letter-spacing: 0.5px;
+    font-weight: 700;
+    margin: 0 0 0 0;
+    /* color: rgb(87, 2, 30); */
+    color: rgb(48, 18, 28);
+    font-weight: bold;
```
Comment (angeliquejw) [inline project1/styles.css:57]: This rule is overriding the other `font-weight` rules on lines 51 and 53.

File: project1/styles.css [hunk 16]
```diff
@@ -0,0 +1,157 @@
...
+}
+
+h3 {
+    font-family: 'PT Sans', sans-serif;
+    font-size: 13px;
+    text-align: center;
+    font-weight: 200;
+    letter-spacing: 0.5px;
+    font-weight: 700;
+    margin: 0 0 0 0;
```
Comment (angeliquejw) [inline project1/styles.css:54]: As previously noted:

```suggestion
    margin: 0;
```

File: project1/styles.css [hunk 17]
```diff
@@ -0,0 +1,157 @@
...
+
+main {
+    background-color: whitesmoke;
+    /* border: 2px solid burlywood; */
+    border: 2px solid pink;
+    padding: 35px;
+    margin: 0 auto;
+}
+
+/* .Hlight {
```
Comment (angeliquejw) [inline project1/styles.css:86]: You ended up not using this class, but remember that classnames should be readable and lowercase; see [my hints on naming classes](https://mica-web.github.io/gd-220/hints/naming-classes). I think this would be useful, too, because the other classnames (like "first" and "one") could also be improved.

File: project1/styles.css [hunk 18]
```diff
@@ -0,0 +1,157 @@
...
+    /* background-color: hsla(102, 34%, 50%, 0.3); */
+/* background-color: hsla(60, 100%, 50%, 0.3); */
+/* } */
+
+.cut {
+    margin-top: -50px;
```
Comment (angeliquejw) [inline project1/styles.css:92]: This element has a negative `margin-top` of 50px, but the first element inside this `div` is your `h1` -- which has a _positive_ `margin-top` of 40px. Any time you look to set a negative margin or padding, first investigate what element might be _creating_ that space.

File: project1/styles.css [hunk 19]
```diff
@@ -0,0 +1,157 @@
...
+}
+
+.two {
+    text-align: center;
+    font-family: 'PT Sans', sans-serif;
+    font-size: 12px;
+    font-style: italic;
+    margin-top: 10px;
+    margin-bottom: 10px;
+    margin: 0 auto;
```
Comment (angeliquejw) [inline project1/styles.css]: This rule is resetting the margins you just set on lines 112-113. If you want zero margins, you can delete lines 112-113 or, if you want them you should _still_ delete them and update this rule:
```suggestion
    margin: 10px auto;
```

File: project1/styles.css [hunk 20]
```diff
@@ -0,0 +1,157 @@
...
+    font-family: 'PT Sans', sans-serif;
+    font-size: 12px;
+    font-style: italic;
+    margin-top: 10px;
+    margin-bottom: 10px;
+    font-weight: bold;
+    background-color: hsla(60, 100%, 50%, 0.3)
+}
+
+.two {
```
Comment (angeliquejw) [inline project1/styles.css]: This rule doesn't appear to be used in your HTML; if that's intentional, you can delete lines 107-117.

File: project1/styles.css [hunk 21]
```diff
@@ -0,0 +1,157 @@
...
+    font-size: 12px;
+    font-style: italic;
+    margin-top: 10px;
+    margin-bottom: 10px;
+    margin: 0 auto;
+    font-weight: bold;
+    background-color: hsla(0, 100%, 71%, 0.4);
+}
+
+.quote {
```
Comment (angeliquejw) [inline project1/styles.css]: This class is also not used in your HTML.

**Review by angeliquejw — APPROVED** [review]
Overall, this looks quite good! Made some notes about bits that have room for improvement, but you definitely understand the core bits of HTML and CSS we've covered in class so far. 👍🏻

Grade to be posted in CANVAS by end of day.

File: project1/index.html (hunk 4 above)
Comment (itseugene) [reply project1/index.html:27]: In 43, I used <b> and also <span>. In this case, how can I change the elements?

Comment (angeliquejw) [reply project1/index.html:27]: If you want to fix this after having made the merge you can either:
1. switch to your master branch, create a new branch, make your project 1 fixes and then merge them into master
1. switch to your master branch, make your project 1 fixes, and push your changes directly to the master branch

Happy to walk you through either of these in class, too!
